tests/cmdline/params/types/test_path.py

```diff
@@ -10,60 +10,43 @@
 """Tests for Path types"""
 
 from aiida.backends.testbase import AiidaTestCase
-from aiida.cmdline.params.types import ImportPath
+from aiida.cmdline.params.types.path import PathURL, _check_timeout_seconds
 
 
-class TestImportPath(AiidaTestCase):
-    """Tests `ImportPath`"""
+class TestPath(AiidaTestCase):
+    """Tests for `PathURL` and `FileURL`"""
 
     def test_default_timeout(self):
         """Test the default timeout_seconds value is correct"""
         from aiida.cmdline.params.types.path import URL_TIMEOUT_SECONDS
 
-        import_path = ImportPath()
+        import_path = PathURL()
 
         self.assertEqual(import_path.timeout_seconds, URL_TIMEOUT_SECONDS)
 
-    def test_valid_timeout(self):
-        """Test a valid timeout_seconds value"""
+    def test_timeout_checks(self):
+        """Test that timeout check handles different values.
 
+         * valid
+         * none
+         * wrong type
+         * outside range
+        """
         valid_values = [42, '42']
 
         for value in valid_values:
-            import_path = ImportPath(timeout_seconds=value)
-
-            self.assertEqual(import_path.timeout_seconds, int(value))
-
-    def test_none_timeout(self):
-        """Test a TypeError is raised when a None value is given for timeout_seconds"""
+            self.assertEqual(_check_timeout_seconds(value), int(value))
 
         with self.assertRaises(TypeError):
-            ImportPath(timeout_seconds=None)
-
-    def test_wrong_type_timeout(self):
-        """Test a TypeError is raised when wrong type is given for timeout_seconds"""
+            _check_timeout_seconds(None)
```

Comment:
If we are simplifying the code and reducing line:
```
for invalid in [None, 'test']:
    with self.assertRaises(TypeError):
        _check_timeout_seconds(invalid)
```

Reply:
sure, will do ;-)